server/controllers/bookController.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable require-jsdoc */
+import models from '../db/models';
+import Utils from '../helpers/utilities';
+
+
+class BookController {
+  static async addBook(req, res) {
+    const {
+      title,
+      description,
+      amount,
+      authorID,
+      status,
+      year
+    } = req.body;
+    try {
+      const book = await models.Books.create({
+        title,
+        description,
+        amount,
+        authorID,
```

Comment:
Are you validating the author's ID is correct?

Reply:
I mean what happens when an ID that doesn't exist is passed?

Reply:
Done! I made it possible to post book without Author

Reply:
That's not the only issue.
Imagine a scenario where a user provides ID "20" for the authorID , and there is no author with such ID. What happens in this case?

http://docs.sequelizejs.com/manual/associations.html